core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -2296,8 +2298,14 @@ public void unparse(
   /** DESCRIPTOR(column_name, ...). */
   public static final SqlOperator DESCRIPTOR = new SqlDescriptorOperator();
 
-  /** TUMBLE as a table-value function. */
-  public static final SqlFunction TUMBLE_TVF = new SqlWindowTableFunction(SqlKind.TUMBLE.name());
+  /** TUMBLE as a table function. */
+  public static final SqlFunction TUMBLE_TVF = new SqlTumbleTableFunction();
+
+  /** HOP as a table function. */
+  public static final SqlFunction HOP_TVF = new SqlHopTableFunction();
+
+  /** SESSION as a table function. */
+  public static final SqlFunction SESSION_TVF = new SqlSessionTableFunction();
 
```

Comment:
What does the `TVF` mean ? How about just name it `HOP_FUN`

Reply:
TVF is the abbrev for "table-valued function".  To clarify, "table-valued function" is the term used in SQL standard. But seems like people in Calcite community like the short version: table function.

I can rename it, but it will be better to use, e.g. HOP_TABLE_FUNCTION. What do you think?

Reply:
I would prefer to name it directly "HOP" and rename the old one to "_HOP", after all, we already rename the group window with prefix "$".

Reply:
How about put this rename in a deprecation process? Directly renaming is not backward compatible. I plan to send a discussion email to community to discuss the deprecation of group window functions. We can decide how to rename it based on the discussion (e.g. if people don't want to deprecate group window functions, it probably means that they want keep `HOP` for hopping by group windowing function).

Reply:
My personal idea is to fully deprecate group window function. HOP,TUMBLE and SESSION will be reserved for table function windowing only.

Reply:
Go ahead and send the mail ~

Reply:
After a second thought, I think I will just make the change about "HOP" and "_HOP" for old and new in this PR, because as you have said, I have already used "$" in the name.


The email discussion I want is about a bigger discussion on if we should fully deprecate group window function. I will do it later and that discussion is not relevant to this PR.

Reply:
There is a naming constraint: 

` Name '_TUMBLE' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. `

So the realistic naming is `TUMBLE_`. Do you think it is a good idea to rename all group window function operator with "_" as suffix?

Reply:
I'm fine with that.

Reply:
Used , for example, `TUMBLE_OLD` as the naming for grouped window functions.